### PR TITLE
Fix ebook.de

### PR DESCRIPTION
**Pull request by stefan-kolb**
Refs #3854

Unfortunately not a very clean fix, but it is fine again.
Using the normal InputStream does not return anything. Probably as it is detected as automated crawler?
The interface with InputStream is just not really fitting here.

### CODE REVIEW

File: src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java [hunk 1]
```diff
@@ -61,8 +63,9 @@ default void doPostCleanup(BibEntry entry) {
             return Optional.empty();
         }
 
-        try (InputStream stream = new BufferedInputStream(getURLForID(identifier).openStream())) {
-            List<BibEntry> fetchedEntries = getParser().parseEntries(stream);
+        try {
+            HttpResponse<String> response = Unirest.get(getURLForID(identifier).toString()).asString();
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java]: Well, this now has the disdvantage that the Stream does not get closed. That's why try(....) is preferable.

**Comment by Siedlerchr** [comment]
The ISBN fetcher tries ebook.de first and if no result is found, it switches to the chimbori/amazon fetcher. 
Is this still valid?

**Comment by stefan-kolb** [comment]
Yes.

File: src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java (hunk 1 above)
Comment (Siedlerchr) [reply src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java]: Ah sorry, got confused by the diff.
Why is this ByteArrayInputStream needed? Is there no way to directly get the response with unirest?

Comment (stefan-kolb) [reply src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java]: Caused by the interface

Comment (tobiasdiez) [reply src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java]: Not sure whats better but we should stick to one: unirest vs our UrlDownloader

Comment (Siedlerchr) [reply src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java]: We could directly use jsoup. We use that in dfiferent cases

Comment (stefan-kolb) [reply src/main/java/org/jabref/logic/importer/IdBasedParserFetcher.java]: Jsoup is for a DOM that needs to be parsed. I don't need that here. I just need the body contents.

**Pull request merged**